content/subscription/details.md

```diff
@@ -4,19 +4,21 @@ description: Learn about Docker subscription tiers and their features
 keywords: subscription, personal, pro, team, business, features
 ---
 
+A Docker subscription allows the use of Docker components including the Docker CLI, Docker Compose, Docker Engine, Docker Desktop, Docker Hub, Kubernetes, Docker Build and Docker BuildKit, Docker Official Images, and more.
```

Comment:
Not really related to your change, but Docker BuildKit isn't a thing and I wouldn't include Kubernetes here either. I think we can shorten it down to this.

```suggestion
A Docker subscription allows the use of Docker components including the Docker CLI, Docker Compose, Docker Engine, Docker Desktop, Docker Hub, Docker Official Images, and more.
```